app/src/main/java/com/infomaniak/mail/ui/main/thread/actions/ActionItemView.kt

```diff
@@ -56,9 +57,7 @@ class ActionItemView @JvmOverloads constructor(
                 val iconHorizontalPadding = getDimenOrNull(R.styleable.ActionItemView_iconPaddingHorizontal)
                 val iconPaddingStart = iconHorizontalPadding ?: getDimenOrNull(R.styleable.ActionItemView_iconPaddingStart)
                 val iconPaddingEnd = iconHorizontalPadding ?: getDimenOrNull(R.styleable.ActionItemView_iconPaddingEnd)
-
-                icon.setMarginsRelative(start = iconPaddingStart)
-                description.setMarginsRelative(end = iconPaddingEnd)
+                container.setPaddingRelative(start = iconPaddingStart, end = iconPaddingEnd)
```

Comment:
Shouldn't the iconPaddingEnd represent the padding between icon and the title instead?